spec/rubocop/cop/rspec/http_status_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::HttpStatus, :config do
+  subject(:cop) { described_class.new(config) }
+
+  context 'when EnforcedStyle is `symbol`' do
+    let(:cop_config) { { 'EnforcedStyle' => 'symbol' } }
+
+    it 'registers an offense when using numeric value' do
+      expect_offense(<<-RUBY)
+        it { is_expected.to have_http_status 200 }
+                            ^^^^^^^^^^^^^^^^^^^^ Prefer `symbol` over `numeric` to describe http status code.
+      RUBY
+    end
+
+    it 'does not register an offense when using symbol value' do
+      expect_no_offenses(<<-RUBY)
+        it { is_expected.to have_http_status :ok }
+      RUBY
+    end
+
+    include_examples 'autocorrect',
+                     'it { is_expected.to have_http_status 200 }',
+                     'it { is_expected.to have_http_status :ok }'
+
+    include_examples 'autocorrect',
+                     'it { is_expected.to have_http_status 404 }',
+                     'it { is_expected.to have_http_status :not_found }'
+  end
+
+  context 'when EnforcedStyle is `numeric`' do
+    let(:cop_config) { { 'EnforcedStyle' => 'numeric' } }
+
+    it 'registers an offense when using symbol value' do
+      expect_offense(<<-RUBY)
+        it { is_expected.to have_http_status :ok }
+                            ^^^^^^^^^^^^^^^^^^^^ Prefer `numeric` over `symbol` to describe http status code.
+      RUBY
+    end
+
+    it 'does not register an offense when using numeric value' do
+      expect_no_offenses(<<-RUBY)
+        it { is_expected.to have_http_status 200 }
+      RUBY
+    end
+
+    include_examples 'autocorrect',
+                     'it { is_expected.to have_http_status :ok }',
```

Comment:
Please check if this will work as expected if parenthesis are used
`it { is_expected.to have_http_status(:ok) }'`